clastix/rancher-extension-clastix
# Fix some issues with Kamaji extension

This PR fixes some issues I've detected while doing tests with Rancher Extensions for the upcoming Rancher 2.9 release:

- a routing issue (hard refresh on product route wouldn’t work - fixes to `product.ts` and `routes.ts`)
- layout issue (sidebar disappears - removal of `layout: "single"` in `Dashboard` component)
- product name needs to be a single word like `kamaji`  not `multitenancy management`
- updated shell package dependency to latest stable version
- fixes some linting issues
- version bumping to prepare for a release


To test:
- check out branch and yarn install dependencies
- do `yarn build-pkg kamaji` + `yarn serve-pkgs` to serve the locally built assets and then go to your Rancher instance and do a developer load
https://extensions.rancher.io/extensions/extensions-getting-started#test-built-extension-by-doing-a-developer-load

**Before**
<img width="1883" alt="before" src="https://github.com/clastix/rancher-extension-clastix/assets/97888974/8235bc67-0309-4c67-bb6b-2ee44f602dd1">

**After**
<img width="1883" alt="after" src="https://github.com/clastix/rancher-extension-clastix/assets/97888974/04f90cd5-02cc-4702-ac0c-0d6c95ea1447">
